src/components/editions/pullquote.tsx

```diff
@@ -0,0 +1,128 @@
+import type { SerializedStyles } from '@emotion/core';
+import { css } from '@emotion/core';
+import { from } from '@guardian/src-foundations/mq';
+import { headline } from '@guardian/src-foundations/typography';
+import { SvgQuote } from '@guardian/src-icons/quote';
+import type { Format, Option } from '@guardian/types';
+import { map, withDefault } from '@guardian/types';
+import { pipe2 } from 'lib';
+import type { FC, ReactNode } from 'react';
+import { basePx, darkModeCss } from 'styles';
+import { getThemeStyles } from 'themeStyles';
+
+export const pullquoteWidth = '10.875rem';
+const pullquoteTailSize = '23px';
+
+const styles = (format: Format): SerializedStyles => {
+	const { kicker, inverted } = getThemeStyles(format.theme);
+	return css`
+		width: ${pullquoteWidth};
+		position: relative;
+		box-sizing: border-box;
+		padding: ${basePx(0, 1, 3, 1)};
+		margin: ${basePx(0.75)} ${basePx(2)}
+			calc(${basePx(2)} + ${pullquoteTailSize}) 0;
+
+		color: ${kicker};
+		border: 1px solid ${kicker};
+		border-top: 12px solid ${kicker};
+		border-bottom: none;
+		${darkModeCss`color: ${inverted};`}
+		${darkModeCss`border: 1px solid ${inverted};`}
+		${darkModeCss`border-top: 12px solid ${inverted};`}
+		${darkModeCss`border-bottom: none`}
```

Comment:
Would it be preferable to do this all in one go?

```suggestion
		${darkModeCss`
		    color: ${inverted};
		    border: 1px solid ${inverted};
		    border-top: 0.75rem solid ${inverted};
		    border-bottom: none;
		`}
```

**Note:** I've also translated the `12px` to `0.75rem` here.

You could also simplify because it's only the _colours_ that need to change in dark mode:

```suggestion
		${darkModeCss`
		    color: ${inverted};
		    border-color: ${inverted};
		`}
```

Reply:
Didn't know you could do this 🙈 .. my css-in-js is so rusty. 
I've removed dark mode however as I have found out that editions don't support dark mode currently.